services/app-web/src/components/FileUpload/FileUpload.tsx

```diff
@@ -298,6 +310,8 @@ export const FileUpload = ({
         addFilesAndUpdateList(files)
     }
 
+    const summary = `${fileItems.length} file${fileItems.length !== 1 ? 's' : ''} selected`
```

Comment:
Not a blocker, and probably more of a pain than it's worth, given its location in app-api, but I recently added a `pluralizer` helper to the codebase that might have saved you one-third of a second.  🙂

Reply:
Do we have any rules around importing across package boundaries? I kinda think it's better to have a lib that both services import, but like you said, that's a lot of work for a couple lines of code.

Reply:
Rules no. Limitations (at least up until a couple months ago) yes. At one point because of using CRA and _not_ having yarn workspaces set up at the beginning we simply weren't able to import from anywhere else in the codebase into app-web. 

That led to weird patterns like the `common-code` folder in app-web which is where we put code we want to share between. That should be the starting place for any lib. I think we are well positioned to that now with the yarn workspaces.  If ya want, Isuggest for now just put code you want to share between there.